src/server/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 project(server)
 
-find_package(Qt6 REQUIRED COMPONENTS Network Xml Svg)
+find_package(Qt6 REQUIRED COMPONENTS Network Svg)
+find_package(Poco 1.12.5 REQUIRED XML)
```

Comment:
Poco version 1.12.5 is still used on Windows, but Linux and MacOS are now using 1.13.2